services/QuillLMS/engines/evidence/spec/models/evidence/automl_model_spec.rb

```diff
@@ -164,16 +164,17 @@ def automl_model.labels_have_associated_rules
       end
 
       it 'should strip any whitespace that is on "automl_model_id" before looking records up via Google' do
-        ENV['AUTOML_GOOGLE_PROJECT_ID'] = 'PROJECT_ID'
```

Comment:
Just a note, setting ENV variables is a global change that doesn't get reset after the test is run, and should be avoided. Assigning to a constant and using `stub_const` is one way to get around this.